google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoManagedVmDeployment.java

```diff
@@ -73,7 +93,10 @@ public void run() {
           loggingHandler,
           "Created temporary staging directory: " + stagingDirectory.getAbsolutePath());
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      logger.error(e);
```

Comment:
yeah, and I just noticed that I missed the execution exception. My head is not on straight this morning.